messaging-experiments.json

```diff
@@ -1116,5 +1116,279 @@
         }
       ]
     }
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-V2-79-RELEASE",
+    "enabled": true,
+    "filter_expression": "env.version >= '79.' && (env.channel == 'default' || env.channel == 'beta')",
```

Comment:
filter expression intentionally allows both beta and default for easier testing on local